whitepapers/ingest/Graph.md

```diff
@@ -0,0 +1,29 @@
+# MutableGraph [NOT IMPLEMENTED]
```

Comment:
I do rather like the interface I ended up with here, which uses an understanding of the schema to vastly simplify implementing regulator logic. I wouldn't be opposed to refactoring it to use networkx internally, assuming that actually results in something simpler to understand/maintain.